src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'ADD_BOOK':
+      return [...state, action.payload];
+    case 'REMOVE_BOOK':
+      return [...state, action.payload];
+    default:
+      return state;
+  }
+};
+
+const addBook = () => ({
+  type: 'ADD_BOOK',
+  payload: {
+    title: 'Harry Potter',
+  },
+});
+
+const removeBook = () => ({
+  type: '',
+  payload: {
+    title: 'Harry Potter',
+  },
+});
+
+export { addBook, removeBook };
```

Comment:
- [ ] in this file you should Define ADD_BOOK and REMOVE_BOOK as action types, please keep in mind that The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)